### PR TITLE
Rename build-rootfs to run-arcade-build-rootfs

### PR DESCRIPTION
**Pull request by am11**
When working with cross build, I found it a bit confusing that `build-rootfs.sh` in this repo is basically a wrapper script which invokes the script with same name from dotnet/arcade repository. The main confusion is the difference in their (positional) arguments.

The purpose of this PR is to disambiguate the names by renaming the script in this repo to <del>`invoke-arcade-rootfs.sh R`</del> `run-arcade-build-rootfs.sh`. <del>If this name sounds strange, I am open for suggestions. :)</del>

### CODE REVIEW

File: src/raspbian/10/helix/arm32v6/hooks/pre-build [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 SCRIPT=$(readlink -f "$0")
 SCRIPTPATH=$(dirname "$SCRIPT")
-$SCRIPTPATH/../../../../../ubuntu/build-scripts/build-rootfs.sh raspbian-10 raspbian armv6 "" "" true
+$SCRIPTPATH/../../../../../ubuntu/build-scripts/invoke-arcade-rootfs.sh raspbian-10 raspbian armv6 "" "" true
```
Comment (mthalman) [inline src/raspbian/10/helix/arm32v6/hooks/pre-build:5]: How about this?
```suggestion
$SCRIPTPATH/../../../../../ubuntu/build-scripts/run-arcade-build-rootfs.sh raspbian-10 raspbian armv6 "" "" true
```

Comment (am11) [reply src/raspbian/10/helix/arm32v6/hooks/pre-build:5]: Much better 👌

**Review by mthalman — APPROVED** [review]

**Comment by am11** [comment]
It looks like when we change pre-build hook, we don't rebuild the image. In this case, if I had a typo, it went undetected by the CI. 🤔

**Comment by mthalman** [comment]
Yep. I just realized that myself. To workaround this, you can make any kind of edit to one of the Dockerfiles, like adding a comment. That will trigger a cache miss and rebuild the image.

**Comment by mthalman** [comment]
Then you can just revert the edit after verifying the build.

**Comment by am11** [comment]
Yup, in this case, we would need to make change in every corresponding Dockerfile to be 100% sure. 🙈

BTW, I used these commands on macOS for mass replacement:

```sh
$ git grep -l build-rootfs.sh ':/*pre-build' |\
     xargs sed -i '' 's/build-rootfs.sh/run-arcade-build-rootfs.sh/g'
$ mv src/ubuntu/build-scripts/{,run-arcade-}build-rootfs.sh
```

So I will change just one.

**Comment by am11** [comment]
It was all green, so I reverted the dummy change. This is ready.

**Pull request merged**